data/v2.0.0/countries-of-birth.json

```diff
@@ -0,0 +1,1010 @@
+[
+   {
+      "en-gb": "England",
+      "cy": "Lloegr"
+   },
+   {
+      "en-gb": "Wales",
+      "cy": "Cymru"
+   },
+   {
+      "en-gb": "Scotland",
+      "cy": "Yr Alban"
+   },
+   {
+      "en-gb": "Northern Ireland",
+      "cy": "Gogledd Iwerddon"
+   },
+   {
+      "en-gb": "Republic of Ireland",
+      "cy": "Gweriniaeth Iwerddon"
+   },
+   {
+      "en-gb": "Afghanistan",
+      "cy": "Affganistan"
+   },
+   {
+      "en-gb": "\u00c5land Islands",
+      "cy": "Ynysoedd \u00c5land"
```

Comment:
The encoding here is not working.

Reply:
In what context is it not working?

Reply:
It's not readable, we should be using utf-8 encoding throughout.

Reply:
Ah, sorry. I see what you mean. I though you meant the unicodes weren't resolving. Yes, dumps to a file defaults to ascii by the looks of it. I've enforced utf-8 explicitly.